src/main/scala/gestalt/Toolbox.scala

```diff
@@ -263,6 +263,16 @@ trait TypeToolbox extends Toolbox { t =>
   def <:<(tp1: Type, tp2: Type): Boolean
   def typeOf(path: String): Type
 
+  val Ascribe: AscribeHelper
+  trait AscribeHelper {
+    def unapply(arg: Tree): Option[(Tree, TypeTree)]
+  }
+
+  val SeqLiteral: SeqLiteralHelper
+  trait SeqLiteralHelper {
+    def unapply(tree: Tree): Option[Seq[Tree]]
+  }
```

Comment:
If it's a varargs call, I'm thinking just flatten the arguments for inspection -- we can special case it in the extractor of `Apply`. According to our last discussion, we loose nothing by doing so.